### PR TITLE
Added other foldable devices representation

### PR DESCRIPTION
**Pull request by ancirja-m**
## 📃 Description
Added other foldable devices representation inside DeviceModel enum

## 🤔 Motivation and Context
In the end, we should write Android Tests for specific devices

## 🧪 How Has This Been Tested?
For now, manually, using the annotations for the existing tests.

## ⚒️ Does the PR contain new tests or refactored old ones?
I changed some olded tests to ignore devices where it's not working.

## 📷 Screenshots (if appropriate)
Not needed.

## 📦 Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Enhancement to a current functionality
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix (non-breaking change which fixes an issue. Please add issue link here too).

## ✅ Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] This PR contains new tests that cover the new code.
- [x] This PR refactor previous tests to cover the new code.
- [ ] This PR is part of a set of PRs that build a bigger feature. If so, please, add here links to previously merged or open PRs.

### CODE REVIEW

File: utils/test-utils/src/main/java/com/microsoft/device/dualscreen/testing/runner/FoldableJUnit4ClassRunner.kt [hunk 1]
```diff
@@ -36,8 +36,8 @@ class FoldableJUnit4ClassRunner : AndroidJUnit4ClassRunner {
         val currentDeviceModel = uiDevice.getDeviceModel()
         val targetDevices: TargetDevices? = getAnnotation(description, TargetDevices::class.java)
         if (targetDevices == null ||
-            targetDevices.devices.any { it == currentDeviceModel } ||
-            !targetDevices.ignoreDevices.any { it == currentDeviceModel }
+            (targetDevices.devices.isNotEmpty() && targetDevices.devices.any { it == currentDeviceModel }) ||
+            (targetDevices.ignoreDevices.isNotEmpty() && !targetDevices.ignoreDevices.any { it == currentDeviceModel })
```
Comment (CesarValiente) [inline utils/test-utils/src/main/java/com/microsoft/device/dualscreen/testing/runner/FoldableJUnit4ClassRunner.kt:40]: These two lines are a bit difficult to read and understand, maybe is a good idea to create their own functions with a name that can be understandable easily and use them?
e.g.
```
if (targetDevices == null ||
   targetDeviceIsCurrentDevice ||
   ignoreDeviceIsNotCurrentDevice) {
   ....
   }
```

Thoughts?

**Review by CesarValiente — APPROVED** [review]
LGTM! I've just left a suggestion Andrei. Please have a look. Otherwise, everything looks good!

File: utils/test-utils/src/main/java/com/microsoft/device/dualscreen/testing/runner/FoldableJUnit4ClassRunner.kt (hunk 1 above)
Comment (ancirja-m) [reply utils/test-utils/src/main/java/com/microsoft/device/dualscreen/testing/runner/FoldableJUnit4ClassRunner.kt]: Done. Added two functions: currentDeviceIsOnTargetDevices, currentDeviceNotInIgnoredDevices

**Review by bimiron — APPROVED** [review]
Looks great!

File: utils/test-utils/src/main/java/com/microsoft/device/dualscreen/testing/runner/FoldableJUnit4ClassRunner.kt (hunk 1 above)
Comment (CesarValiente) [reply utils/test-utils/src/main/java/com/microsoft/device/dualscreen/testing/runner/FoldableJUnit4ClassRunner.kt]: Awesome! thanks Andrei! It looks great now.

**Pull request merged**